specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -1210,6 +1210,14 @@
           "description": "Password for authentication.",
           "$ref": "../datafactory.json#/definitions/SecretBase"
         },
+        "packageCollection": {
+          "type": "object",
+          "description": "Under where packages are created when querying database. Type: string (or Expression with resultType string)."
+        },
+        "certificateCommonName": {
+          "type": "object",
+          "description": "Certificate Common Name when TLS is enabled. Type: string (or Expression with resultType string)."
+        },
```

Comment:
Adding properties to a stable API version is typically regarded as a breaking change, since older versions of the client SDKs will be unable to read them. What is the reason for this change?

Reply:
Although they are new properties but the older versions of the client SDKs still be able to read them. It is just a simple update with some new features.

Reply:
Make sure GET-PUT pipeline is good for old client. I.e., if new param is defined in resource, but old client GET the resource (not able to parse the new param), then PUT it back, it will not erase that new param previous defined.

https://microsoft.sharepoint.com/:w:/r/teams/azureresourcemanagerteam/_layouts/15/guestaccess.aspx?rtime=oPAFQY8Z10g&share=EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw